presidio-anonymizer/app.py

```diff
@@ -25,9 +28,11 @@ def anonymize():
                 data = AnonymizerRequest(content)
                 text = AnonymizerEngine().anonymize(data)
             except InvalidParamException as e:
-                return e.err, 400
-            except Exception:
-                # TODO add logger 2652
+                logging.warning(
+                    {f"failed to anonymize text with validation error: {e.err_msg}"})
```

Comment:
The message here is unique to the issue. I don't think we need the entire trace. 
It will seem as an internal exception instead of an error of the user usage.

Reply:
I'm used to taking the trace and paste it, instead of just looking for strings, can show me the whole callstack as well. Your choice, its for us so I don't see why not being more verbose when we debug issues.

Reply:
I don't think exceptions should be in debug mode when it's not really an exception but a user error.

Reply:
There are static code analysis tools that might actually fail in that case, its considered best practice to reserve the exception, even if its info:
https://jira.sonarsource.com/browse/RSPEC-1166